flink-libraries/flink-table/src/main/scala/org/apache/flink/table/api/table.scala

```diff
@@ -726,7 +726,10 @@ class Table(
     * }}}
     *
     * @param offset number of records to skip
+    *
+    * @deprecated Please use [[Table.offset()]] and [[Table.fetch()]] instead.
     */
+  @deprecated(message = "Deprecated in favor of Table.offset() and Table.fetch()", since = "1.4.0")
```

Comment:
This kind of deprecation does not work for Java. Please add the Java annotation as well.